srcs/app/server/Location.cpp

```diff
@@ -0,0 +1,4 @@
+#include <Location.hpp>
+
+Location::Location(const CommonConfig &common) : common(common) {
```

Comment:
I think this file can go in  `srcs/app/config/` because it's a component of the `ServerConfig` class

Reply:
You are completely right! I will fix it for sure, thanks!!